install-server.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -e
+
+echo -e "\n\nEverything is now getting setup. This process will take a few minutes...\n\n"
+
+sudo apt update && sudo apt install gpg jq -y && sudo apt clean
```

Comment:
can't we install these via nix?

Reply:
We can, these might not actually be required for the script, so I will remove them and see.